Remotion/Data/DomainObjects.ObjectBinding.IntegrationTests/Database/CreateDB.sql

```diff
@@ -8,16 +8,6 @@ END
 GO
 
 CREATE DATABASE DBPrefix_RemotionDataDomainObjectsObjectBindingIntegrationTestDomain
-ON PRIMARY (
```

Comment:
Please add a note to the jira ticket that we have rmeoved the database location / no longer persist the database outside of the docker image in case we ever run across this topic again.